package.json

```diff
@@ -18,6 +18,7 @@
     "bazel:buildifier": "find . -type f \\( -name \"*.bzl\" -or -name WORKSPACE -or -name BUILD -or -name BUILD.bazel \\) ! -path \"*/node_modules/*\" | xargs buildifier -v --warnings=attr-cfg,attr-license,attr-non-empty,attr-output-default,attr-single-file,constant-glob,ctx-args,depset-iteration,depset-union,dict-concatenation,duplicated-name,filetype,git-repository,http-archive,integer-division,load,load-on-top,native-build,native-package,output-group,package-name,package-on-top,redefined-variable,repository-name,same-origin-load,string-iteration,unused-variable,unsorted-dict-items,out-of-order-load",
     "bazel:format-lint": "yarn -s bazel:buildifier --lint=warn --mode=check",
     "dev-app": "ibazel run //src/dev-app:devserver",
+    "dev-app-proxy": "browser-sync start --proxy localhost:4200 --host 0.0.0.0 --port 4201 --no-open --no-ui --no-watch",
```

Comment:
Should we also disable ghost mode? This was one of the points you made when we had browser-sync for our dev-app as it could throw of change detection?

Reply:
Good point, I'd forgotten about that 👍. Added.